src/base/global.h

```diff
@@ -34,15 +34,13 @@
 const char C_TORRENT_FILE_EXTENSION[] = ".torrent";
 
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 7, 0)
 template <typename T>
-constexpr typename std::add_const<T>::type &qAsConst(T &t) noexcept { return t; }
+constexpr typename std::add_const<T>::type &asConst(T &t) noexcept { return t; }
 
 // prevent rvalue arguments:
 template <typename T>
-void qAsConst(const T &&) = delete;
-#endif
+void asConst(const T &&) = delete;
```

Comment:
I'm not entirely sure what's going on here to be honest. Is this still needed?

Reply:
I would move this deleted template to the end.

Reply:
Done.